cli/functionary/environment.py

```diff
@@ -42,57 +39,51 @@ def get_environment_list():
 @click.group("environment")
 @click.pass_context
 def environment_cmd(ctx):
+    """
+    Click group to associate all environment related commands with
+    """
     pass
 
 
 @environment_cmd.command()
 @click.pass_context
 def set(ctx):
     """
-    Command to set the environment id based on user input
-
-    Args:
-        Ctx: The click context
-
-    Returns:
-        None
-
-    Raises:
-        ClickException if bad environment number
+    Set the active environment
     """
-    env_list = get_environment_list()
+    env_list = _get_environment_list()
     index = 1
     click.echo("Available Environments:")
     for item in env_list:
-        click.echo(f"    {index}) {item.get('name')}")
+        click.echo(f"    {index}) {item.get('name')} - {item.get('team')}")
```

Comment:
This displays more cleanly with the team name listed first, then the name.  It ends up looking something like:

```
Available Environments:
    1) beergarden - default
    2) beergarden - dev
    3) beergarden - prod
    4) foxhound - default
Select environment:
```

Which just looks cleaner with the team names nicely lined up.